### PR TITLE
Feature/e UI 6924

### PR DESCRIPTION
**Pull request by MateiHMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-6924

### Change description ###
Open document in a new tab

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by johnbenjamin-hmcts — APPROVED** [review]

**Review by tarikdogan-hmcts — APPROVED** [review]

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/components/case-file-view-folder/case-file-view-folder.component.ts [hunk 1]
```diff
@@ -46,7 +49,11 @@ export class CaseFileViewFolderComponent implements OnInit, OnDestroy {
     }
   }
 
-  constructor() {
+  constructor(
+    private windowService: WindowService,
```
Comment (Daniel-Lam) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/components/case-file-view-folder/case-file-view-folder.component.ts:53]: @MateiHMCTS - can these two be `readonly` like `documentManagementService`?

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/components/case-file-view-folder/case-file-view-folder.component.ts [hunk 2]
```diff
@@ -169,29 +210,6 @@ export class CaseFileViewFolderComponent implements OnInit, OnDestroy {
...
-  public sortDataSourceDescAlphabetically() {
-    const sortedData = this.nestedDataSource.map(item => {
-      item.sortChildrenDescending();
-      return item;
-    });
-
-    this.updateNodeData(sortedData);
-  }
-
   public updateNodeData(data: DocumentTreeNode[]) {
```
Comment (Daniel-Lam) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/components/case-file-view-folder/case-file-view-folder.component.ts:213]: @MateiHMCTS - consider setting a return type for this function, even if it is `void`

**Review by Daniel-Lam — CHANGES_REQUESTED** [review]
@MateiHMCTS - just a few small changes to look at

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/components/case-file-view-folder/case-file-view-folder.component.ts [hunk 3]
```diff
@@ -128,6 +140,24 @@ export class CaseFileViewFolderComponent implements OnInit, OnDestroy {
     return uncategorisedNode;
   }
 
+  public sortDataSourceAscAlphabetically() {
```
[the remark this answers is not in the document]
Comment (Daniel-Lam) [reply projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/components/case-file-view-folder/case-file-view-folder.component.ts:143]: @MateiHMCTS - still missing return type

**Review by Daniel-Lam — CHANGES_REQUESTED** [review]
@MateiHMCTS - you missed the return type for one: `sortDataSourceAscAlphabetically()`

**Review by Daniel-Lam — APPROVED** [review]

**Pull request merged**